src/Plugin.php

```diff
@@ -137,34 +164,57 @@ public function copyWpConfig( PackageEvent $event ) {
 		$wordpressInstallDir = $installationManager->getInstallPath( $wordpressPackage );
 
 		if ( ! is_dir( $wordpressInstallDir ) ) {
-			$this->io->write( '<warning>The installation path of WordPress seems to be broken. wp-config.php not copied.</warning>' );
+			$this->io->writeError( '<warning>The installation path of WordPress seems to be broken. wp-config.php not copied.</warning>' );
```

Comment:
`writeError()` is also used by Composer to make sure the output is printed to STDERR.